io/src/main/scala/sbt/io/Hash.scala

```diff
@@ -52,7 +53,7 @@ object Hash {
   def apply(as: Array[Byte]): Array[Byte] = apply(new ByteArrayInputStream(as))
 
   /** Calculates the SHA-1 hash of the given file.*/
-  def apply(file: File): Array[Byte] = Using.fileInputStream(file)(apply)
+  def apply(file: File): Array[Byte] = Files.readAllBytes(file.toPath)
```

Comment:
Wat. This reads all the bytes in the file. It doesn't calculate the SHA-1 hash for it..

Reply:
well then apply the hash to it afterwards... the slow bit is the `FileFilterStream`

like I said, I have absolutely no way of validating anything about this PR.

Reply:
Ok.

Reply:
```scala
  def apply(file: File): Array[Byte] = apply(Files.readAllBytes(file.toPath))
```